### DISCUSSION TITLE
GD Library extension not available with this PHP installation. ("v8.0.0")

### DISCUSSION THREAD

**Discussion by rhoyle** [spatie/laravel-medialibrary#2547] [Q&A]
Intervention\Image\Exception\NotSupportedException
GD Library extension not available with this PHP installation.

I am using Laravel v8.49.2, with php v8.0.0 Zend Engine v4.0.0-dev

C:\xampp\htdocs\ironman\vendor\intervention\image\src\Intervention\Image\Gd\Driver.php:19

**Comment by Anticom** [comment]
What does `php -m` give you? You should have `gd` in the list of extensions. Otherwise your GD Extension just isn't set up correctly.

**Reply by pradeepCodoffer** [reply]
I don't have gd in my php -m list . And i'm using laravel 10 . So how can i setup this extention in this version

**Reply by parallels999** [reply]
add it

**Comment by madfortech** [comment]
Intervention
 \ 
Image
 \ 
Exception
 \ 
NotSupportedException
PHP 8.1.17
10.48.22
GD Library extension not available with this PHP installation.
My DB_CONNECTION=sqlite 
any one help me
